packages/rum/test/benchmarks/analyzer.js

```diff
@@ -192,6 +192,13 @@ function calculateResults(resultMap) {
          */
         result[metricName] = value
       } else {
+        /**
+         * Skip if the value inside the array is null
+         * which means the metric type is not captured inside the browser
+         */
+        if (value[0] == null) {
+          return
```

Comment:
Is there an indication of the metrics that aren't captured for a particular browser.

Reply:
yes, I have added a comment there in prev line. want me to improve it?